app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -253,6 +253,22 @@ data class Patient(
         scheduled: Appointment.Status = Appointment.Status.SCHEDULED
     ): Flowable<Boolean>
 
+    @Query("""
+        SELECT (
+            CASE
+                WHEN (COUNT(uuid) > 0) THEN 1
+                ELSE 0
+            END
+        )
+        FROM Patient
+        WHERE updatedAt >= :instantToCompare AND syncStatus = :pendingStatus AND uuid = :patientUuid
```

Comment:
I am not sure why we are checking `updatedAt >= :instantToCompare` with and `equal to`.
This won't really make a difference to the logic because this time won't be the same in any case but it does add some confusion as to why this was done.
Can we remove `=` from all the queries?

Reply:
You're right, I'll do this.